scripts/dns_rr_configs.py

```diff
@@ -0,0 +1,257 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+
+#################################################
+# Script to generate dns round robin config files
+#################################################
+
+import argparse
+import sys
+import os
+from typing import List
+
+def chk_ip(ip: str) -> bool:
+    octs = ip.split(".")
+    if len(octs) != 4:
+        return False
+    for o in octs:
+        try:
+            ioct = int(o)
+            if ioct > 255 or ioct < 0:
+                return False
+        except ValueError:
+            return False
+    return True
+
+def chk_dir(d: str) -> bool:
+    return os.path.isdir(d) and os.access(d, os.W_OK)
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        prog=sys.argv[0],
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+        description="""Creates configuration files for BIND to round robin IPs.""",
+        epilog=f"""Generated configs should be copied to
+name.conf -> /etc/
+forward.<zone> -> /var/named/
+reverse.<zone> -> /var/named/
+
+
+Examples
+To generate configs with default params: zone seagate.com and services s3, iam
+        #> {sys.argv[0]} --ip 192.168.1.0 192.168.1.1
+
+To generate configs for different zones and services
+        #> {sys.argv[0]} --ip 192.168.1.0 192.168.1.1 --zone test.zone --services srv1 srv2
+
+To install BIND use following command
+        #> yum install bind bind-utils
+
+To run BIND use following commands
+        #> systemctl enable named.service
+        #> systemctl start named.service
+
+To restart BIND after config update use following command
+        #> systemctl restart named.service
+        """)
```

Comment:
Either add here to "help", or create separate readme file which explains all needed steps in right order -- e.g.

yum install -y bind bind-utils
systemctl stop named.service # in case it is already started
./script.py -i ip1 ip2 ...
systemctl enable named.service
systemctl start named.service

Goal -- there must be a step-by-step manual, which can be just executed without prior knowledge of internals.

Reply:
removed BIND management commands from help

step-by-step manual is already written and could be found here
docs/LB_and_HA/dns_round_robin_config.md